lerna.json

```diff
@@ -2,6 +2,9 @@
   "npmClient": "yarn",
   "version": "independent",
   "command": {
+    "publish": {
+      "ignoreChanges": ["*.md", "*.spec.js", "**/__snapshots__/**", "styleguide.config.js"]
```

Comment:
note, this is actually a `version` option (`publish` is a soon-to-deprecate alias). However, the (not super great) docs recommend applying `ignoreChanges` to the config root. https://github.com/lerna/lerna/blob/master/commands/version/README.md#--ignore-changes

Reply:
Also, is this right for `README.md` which is part of a package and should probably affect a version patch if changed.

Reply:
> This option is best specified as root lerna.json configuration, both to avoid premature shell evaluation of the globs and to share the config with lerna diff and lerna changed:

Great points. Added.

> Also, is this right for README.md

I have mixed feelings about this. If it's just a markdown change this is _technically_ a documentation change. If we need to publish we can do a force tag locally with the `--no-ignore-changes`? Not sure how often this will be

Reply:
Yup. Fine w me.